colin-api/src/colin_api/resources/v1/corporations.py

```diff
@@ -120,6 +122,42 @@ def request_colin(corp_num: str):  # pylint: disable=too-many-locals, too-many-b
     return jsonify(corp_details_dict), 200
 
 
+@bp.route('/business/<string:corp_num>', methods=['GET', 'OPTIONS'])
+@cross_origin(origin='*')
+def business_request_colin(corp_num: str):
+    """Get business details from COLIN."""
+    corp_num_sql = "\'" + corp_num + "\'"
+    incorp_info_sql = Methods.build_incorp_info_sql(corp_num_sql)
+
+    try:
+        business_info_dict = nro.get_business_info_by_corp_num(corp_num=corp_num_sql)
+        if not business_info_dict:
+            return jsonify({'message': 'Error: Could not find corporation details'}), 404
+
+        incorp_name, incorp_state = Methods.get_incorp_info(incorp_info_sql)
```

Comment:
This returns no results for a valid corp num.

I'm also not sure what the code above(`incorp_info_sql = Methods.build_incorp_info_sql(corp_num_sql)`) does to build the sql this function call uses.  If it's incorrect, please update accordingly.

I think an alternative approach is that you can also just extend the query in `business_info_dict = nro.get_business_info_by_corp_num(corp_num=corp_num_sql)` to include the business state and the legal name.

Reply:
Hi.

I am able to get the response`(state and legal_name)` from `
<img width="1030" alt="Screenshot 2023-09-21 at 8 59 33 AM" src="https://github.com/bcgov/namex/assets/54437031/90f10813-8af4-4e97-b6a5-63d721e21e40">
incorp_info_sql = Methods.build_incorp_info_sql(corp_num_sql)`
When I worked on Lear-Colin-Api, there are some complex logic for getting legal_name. So I used this

Reply:
try 0000096

Reply:
Oops. The error